bin/remove-outputs

```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
+
+sudo rm working_dir/50_Outputs/TestPortfolio_Input/ -rf
```

Comment:
maybe `TestPortfolio_Input` should be parameterized somehow since that's an arbitrary name?

Reply:
Agree (https://github.com/2DegreesInvesting/PACTA_analysis/issues/176)